examples/benchmark/tls_bench.c

```diff
@@ -577,7 +617,7 @@ int bench_tls(void)
     char *cipher, *next_cipher, ciphers[4096];
 
 #ifdef DEBUG_WOLFSSL
-    wolfSSL_Debugging_ON();
+    //wolfSSL_Debugging_ON();
```

Comment:
Should this be commented out?

Reply:
The log spew is pretty bad with debug on, but I will revert this. As a benchmark it makes sense to build without debug when measuring.